typings/@chainlink/index.d.ts

```diff
@@ -26,7 +26,7 @@ declare module '@chainlink/types' {
     network?: string
     returnRejectedPromiseOnError?: Boolean
     verbose?: boolean
-    api: Partial<AxiosRequestConfig>
+    api?: Partial<AxiosRequestConfig>
```

Comment:
Config is super strict, we either leave it flexible with optional api, or extend defaultConfig on every adapter